src/components/EntityInfo/EntityMenu.vue

```diff
@@ -250,7 +258,16 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
   }
 
   get enableAgmLocationChg (): boolean {
-    return !!GetFeatureFlag('enable-agm-location-chg')
+    return !!GetFeatureFlag('supported-agm-location-chg-entities')
```

Comment:
This isn't a boolean FF. You need to check if the current entity type is in the FF string.